content/coverpage.tex

```diff
@@ -72,6 +72,7 @@ \subsection*{\openshmem 1.6}
 \item Robert Kierski, \ac{HPE}
 \item Bryant Lam, \ac{DoD}
 \item Akhil Langer, NVIDIA
+\item Jens Manser, \ac{DoD}
 \item Tiffany M. Mintz, \ac{ORNL}
 \item Bryan Morgan, Intel
 \item William Okuno\footnotemark[2], \ac{HPE}
```

Comment:
@ronawho, please add "Matthew Davis, Georgia Tech" (@davis-matthew), who helped us fix the typos I reported in #545 and #540.

Reply:
Done in https://github.com/openshmem-org/specification/pull/552, thanks for the heads up.

Reply:
Thank you!